localstack/services/awslambda/lambda_api.py

```diff
@@ -152,12 +152,12 @@ def add_function_mapping(lambda_name, lambda_handler, lambda_cwd=None):
     arn_to_lambda[arn].cwd = lambda_cwd
 
 
-def add_event_source(function_name, source_arn, enabled):
+def add_event_source(function_name, source_arn, enabled, batch_size=10):
```

Comment:
Instead of encoding the default value in the function signature, I'd rather handle the default case in the function body (we can define `DEFAULT_BATCH_SIZE` towards top of the file):
```
DEFAULT_BATCH_SIZE = 10

...

def add_event_source(function_name, source_arn, enabled, batch_size=None): 
    batch_size = batch_size or DEFAULT_BATCH_SIZE
    ...
```